.buildkite/publish/docker-compose.yml

```diff
@@ -38,7 +38,10 @@ services:
       # Functional test URLS
       - TEST_FUNCTIONAL_POSTGRES_URI=postgres://prisma:prisma@postgres:5432/PRISMA_DB_NAME
       - TEST_FUNCTIONAL_MYSQL_URI=mysql://root:root@mysql:3306/PRISMA_DB_NAME
-      - TEST_FUNCTIONAL_VITESS_8_URI=mysql://root:root@localhost:33807/PRISMA_DB_NAME
+      - TEST_FUNCTIONAL_VITESS_8_URI=mysql://root:root@vitess-8:33807/PRISMA_DB_NAME
+      # Currently skipped because it failed with
```

Comment:
PR/issue to fix that?

Reply:
It might be due to timeout, https://github.com/prisma/prisma/pull/17704, so I'll try to enable it again after it's merged

Reply:
New PR for unskipping: https://github.com/prisma/prisma/pull/17708